doc/lsst.ip.diffim/tasks/lsst.ip.diffim.AlardLuptonPreconvolveSubtractTask.rst

```diff
@@ -0,0 +1,63 @@
+.. lsst-task-topic:: lsst.ip.diffim.AlardLuptonPreconvolveSubtractTask
+
+##################################
+AlardLuptonPreconvolveSubtractTask
+##################################
+
+AlardLuptonPreconvolveSubtractTask is a modification of AlardLuptonSubtractTask behavior, and it performs image differencing between the `template` image and a custom kernel cross-correlated `science` image.
+
+.. _lsst.ip.diffim.AlardLuptonPreconvolveSubtractTask-description:
+
+Description
+===========
+
+This task takes the pair of science and template exposures, together with the science image source catalog, and produces difference and matched template exposures as outputs.
+
+The process involves selecting sources from the science exposure source catalog, obtaining the convolution kernel, applying this to the `target` image, and subtracting the images.
+
+As discussed in the general package overview the task has two possible modes: one convolving the `template` image and the alternative convolving the `science` image.
+Both modes share the preparation sub-tasks, but the method that performs the convolution is not the same.
+The parameter that controls this behavior is ``mode`` (check `~lsst.ip.diffim.AlardLuptonSubtractConfig`), and it allows ``convolveScience``, ``convolveTemplate`` and finally ``auto`` which means that the decision is made at runtime automatically.
+
+Finally, mask planes are set on the difference and matched template outputs based on the masks of the input science and template images.
```

Comment:
This description doesn't sound right for the preconvolution task. It should state that it creates a Score image, and we don't use mode.

Reply:
Wow, that's a mistake! Thanks!

Reply:
Well, I think I somehow overwrote the changes that I made for this page. I did write a good explanation. I am afraid it is lost.